CHANGES.rst

```diff
@@ -40,6 +40,8 @@ Bug fixes:
   content
   [datakurre]
 
+- Remove references to Products.CMFDefault on meta.zcml
```

Comment:
according the [styleguide](https://docs.plone.org/develop/styleguide/documentation.html#tracking-changes), the changelog entry must be "on top of the current development changes block".